src/7-to-8/cheat-sheet.rst

```diff
@@ -343,3 +342,19 @@ Insert a task:
 
          cylc insert
      - Tasks are inserted automatically when you "trigger" or "set" them.
+
+Re-run a group of tasks in order:
+
+.. list-table::
+   :class: grid-table
+
+   * - **Cylc 7**
+     - **Cylc 8**
+   * - ::
+
+         # reset the task to waiting, they will then re-run in order
+         cylc reset -s=waiting cycle1/task1 cycle2/task2 ...
```

Comment:
In the interests of promoting Cylc 8 as a better way of doing it, let's not gloss over the limitations of the Cylc 7 way. 

```suggestion
         # reset the tasks to waiting, they will then re-run in order
         # (if any have left the task pool, `cylc insert` them back in first)
         cylc reset -s=waiting cycle1/task1 cycle2/task2 ...
```

Reply:
Oh, go on then!